pkg/ccl/backupccl/show.go

```diff
@@ -303,36 +304,45 @@ func showBackupPlanHook(
 				KMSInfo: defaultKMSInfo,
 			}
 		}
-		var incPaths []string
-		incStore := store
-		if incDest, ok := opts[backupOptIncStorage]; ok {
-			if subdir != "" {
-				parsed, err := url.Parse(incDest)
-				if err != nil {
-					return err
-				}
-				parsed.Path = path.Join(parsed.Path, subdir)
-				incDest = parsed.String()
-			}
-			incStore, err = p.ExecCfg().DistSQLSrv.ExternalStorageFromURI(ctx, incDest, p.User())
-			if err != nil {
-				return errors.Wrapf(err, "make incremental storage")
-			}
-			defer incStore.Close()
+		explicitIncPaths := make([]string, 0)
+		explicitIncPath := opts[backupOptIncStorage]
+		if len(explicitIncPath) > 0 {
+			explicitIncPaths = append(explicitIncPaths, explicitIncPath)
 		}
-		incPaths, err = FindPriorBackups(ctx, incStore, IncludeManifest)
-
+		incLocations, err := resolveIncrementalsBackupLocation(
+			ctx,
+			p.User(),
+			p.ExecCfg(),
+			explicitIncPaths,
+			[]string{dest},
+			subdir,
+		)
+		incPaths := []string{}
+		var incStore cloud.ExternalStorage
```

Comment:
incStore was previously being set to `store`. Just want to confirm that sending an unitialized `incStore` below doesn't cause issues.

Reply:
Yes - in the olden days, all the incrementals were in the base directory, so `store` was correct. Now, they may be, or may not, so `incStore` is set with a bit more complexity.